packages/amplify_storage_s3/example/.gitignore

```diff
@@ -41,3 +41,22 @@ app.*.map.json
 
 # Exceptions to above rules.
 !/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
+
+#amplify
+amplify/\#current-cloud-backend
+amplify/.config/local-*
+amplify/logs
+amplify/mock-data
+amplify/backend/amplify-meta.json
+amplify/backend/awscloudformation
+amplify/backend/.temp
+build/
+dist/
+node_modules/
+aws-exports.js
+awsconfiguration.json
+amplifyconfiguration.json
+amplify-build-config.json
+amplify-gradle-config.json
+amplifytools.xcconfig
+.secret-*
```

Comment:
I'm not aware of any enforcement we have for that.  Where in Github did you see that?

Reply:
![image](https://user-images.githubusercontent.com/5827964/118038602-2e2fa100-b324-11eb-8ba6-fa4a288bc886.png)
Just when reviewing changed files. I know it's kind of been widely accepted convention to have it (due to relics which had issues with files not being newline terminated) and it's no big deal if we don't have it. I just noticed it because github pointed it out. It's _probably_ good practice but I don't feel strongly about it either way

Reply:
Small nit, but I support adding blank newlines at end of file simply for the fact that it helps avoid conflicts when adding new stuff below because the new content would not count as a modification to line 62.

Reply:
okay sounds good, added a newline to the gitignore